.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+      command: bundle exec rubocop --parallel --format github
+
+  test_foreman:
+    name: Test against Foreman
+    needs: rubocop
+    strategy:
+      fail-fast: false
+      matrix:
+        foreman:
+          - develop
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
```

Comment:
`foreman_plugin.yml` is for testing Foreman plugins, but `theforeman-rubocop` is not a Foreman plugin.

You'd need to check out Foreman and replace the `theforeman-rubocop` dependency with the PR people are opening.

Reply:
Yes I see we don't have the plugin structure in this case, I'm adjusting the workflow.